src/test/java/org/apache/sling/servlets/get/impl/helpers/JsonRendererTest.java

```diff
@@ -182,7 +184,12 @@ public void testNumber() throws IOException {
     public void testBooleansNoTidy() throws IOException {
         context.currentResource("/content/booleans");
         final String expected = "{\"b2\":false,\"jcr:primaryType\":\"nt:unstructured\",\"s1\":\"true\",\"b1\":true,\"s2\":\"false\"}";
-        assertEquals(expected, getJsonFromRequestResponse());
+        JsonReader jsonReader = Json.createReader(new StringReader(expected));
```

Comment:
please use try with resources here.

Reply:
@priyanka-28 You misunderstood, try with resources calls close automatically, see here for the syntax: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html. No need to catch any exceptions!